examples/Learning_example/learning_example.ipynb

```diff
@@ -0,0 +1,2295 @@
+{
```

Comment:
<p>a plot or picture of their orbits may be nice to illustrate?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aidotse/PASEOS/pull/107/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>What is the easiest way to plot the orbits of an actor?</p>

Reply:
<p>I'd suggest to create orbits with get_constellation, then the user can on the fly change the orbits if they want and you can use the pykep planets directly to plot.</p><p> </p><pre class="ql-syntax" spellcheck="false">from get_constellation import get_constellation

altitude = 550 * 1000 # Constellation attitude above the Earth's ground

inclination = 30.0 # inclination of each orbital plane

nPlanes = 4 # the number of orbital planes (see linked wiki article)

nSats = 4 # the number of satellites per orbital plane

t0 = pk.epoch_from_string("2022-Oct-27 21:00:00") # the starting date of our simulation




# Compute orbits of LEO satellites

planet_list,sats_pos_and_v = get_constellation(altitude,inclination,nSats,nPlanes,t0)




fig = plt.figure(figsize=(6,6),dpi=100)

ax = plt.axes(projection='3d')

colors = sns.color_palette("Paired")

for i in range (nPlanes*nSats):

&nbsp; &nbsp; color_idx = i // nPlanes

&nbsp; &nbsp; pk.orbit_plots.plot_planet(planet_list[i],axes=ax,s=64,color=colors[color_idx])
</pre><p> </p><p>to get opposing directions you will need to multiply the v of the second sat by -1 but in the orbit plot direction isn't show so doesn't matter here.</p>

Reply:
<p>(PS: the current edition also returns orbital period)</p>